src/components/Tabs/Tab/Tab.jsx

```diff
@@ -21,7 +38,7 @@ const Tab = forwardRef(
           ariaHidden={true}
           iconType={iconType}
           icon={icon}
-          className={cx("tab-icon", iconSide)}
+          className={cx(styles.tabIcon, "tab-icon", iconSide)}
```

Comment:
We need to decide if we want to keep the old css classes or not @orrgottlieb @SergeyRoyt

Reply:
Let's wait for Orr and decide

Reply:
Orr suggestion was to keep old classes everywhere for now, complete the migration and then check whether removing old classname breaks something or not - if not then remove